src/Modules/Service Fabric/GE Custom Communications Stack for Service Fabric/GECustomCommunicationsStack/TrinityCommunictionListener.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Fabric;
+using System.Net;
+using System.ServiceModel.Channels;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ServiceFabric.Services.Communication.Runtime;
+using Trinity;
+using Trinity.Network;
+
+namespace GE.ServiceFabric.Services.Communiction.Trinity.Runtime
+{
+
+    /// <summary>
+    /// A Microsoft Trinity Communiactions based listener for Service Fabric based stateless and stateful
+    /// service.
+    /// </summary>
+    public class TrinityCommunictionListener<TServiceContract> : TrinityServer, ICommunicationListener
```

Comment:
@tavitruman let's clean this up and rebase to the dynamic cluster :)

Reply:
@yatli Okay will do and I'm going to work against the Listeners you have in place but make it so that its extensible from the GraphEngineServiceBase.

Reply:
@TaviTruman good to know! I was also wondering how does a developer kick off based on what we have. 👍